README.md

```diff
@@ -0,0 +1,133 @@
+# Plover
+
+> Bringing stenography to everyone.
+
+[![Linux CI badge][]][Linux CI] [![macOS CI badge][]][macOS CI] [![Windows CI badge][]][Windows CI]
+
+|              |              |          |          |                  |                  |
```

Comment:
I overlooked that there is an extraneous table row here, it only become apparent when rendered on GitHub.

It could be a bug in whatever tool used to convert the format (if it was not done manually)...

Reply:
Good catch, fixed in #1210. (For reference I used `pandoc -f rst -t gfm --wrap=preserve --reference-links` for the initial conversion).